lib/iris/tests/unit/_shapefiles/test_transform_geometry.py

```diff
@@ -70,13 +72,21 @@ def test_transform_geometry(
     input_cube_crs,
     output_expected_geometry,
 ):
-    # Assert that all invalid geometries raise the expected error
+    # Check PROJ network settings and disable network access for the test
+    default_pyproj_network = pyproj.network.is_network_enabled()
+    if default_pyproj_network:
+        pyproj.network.set_network_enabled(active=False)
+
     out_geometry = _transform_geometry(
         geometry=input_geometry, geometry_crs=wgs84_crs, cube_crs=input_cube_crs
     )
     assert isinstance(out_geometry, shapely.geometry.base.BaseGeometry)
     assert output_expected_geometry == out_geometry
 
+    # Reset PROJ network settings to default state
+    if default_pyproj_network:
+        pyproj.network.set_network_enabled(active=True)
+
```

Comment:
I'm not clear why this test temporarily disables PROJ network functionality. We can't guarantee this will be the user's true experience.